tests/test_syncer.py

```diff
@@ -294,7 +294,7 @@ def test_bad_syncer(self, mocked_sg):
         correctly handled.
         """
         # Bad setup should raise an exception
-        self.assertRaisesRegexp(
+        self.assertRaisesRegex(
```

Comment:
Gave warnings when running the tests with pytest due to these method being deprecated.